### ISSUE TITLE
new format for Raia

### ISSUE THREAD

**Issue by LeonardSchmiester** [LeonardSchmiester/Benchmark-Models#4]

**Comment by LeonardSchmiester** [comment]
sigmas are not included yet

**Comment by dweindl** [comment]
```
libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_CD274mRNA' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_IL13_cell' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_RecSurf' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_SOCS3' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_SOCS3mRNA' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_pIL4Ra' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_pJAK2' should have a constant value of 'false'.

libSBML Error (SBML component consistency): Any <compartment>, <species>, <parameter>, or (in Level 3) <speciesReference> whose identifier is the value of a 'variable' attribute in an <assignmentRule>, must have a value of 'false' for 'constant'.
Reference: L2V4 Section 4.11.3
 The parameter with id 'sigma_pSTAT5' should have a constant value of 'false'.

```

**Comment by dweindl** [comment]
Observable `scaling_observable_IL13_cell * (IL13_Rec + IL13_DecoyR + p_IL13_Rec + p_IL13_Rec_i)` does not have any parameter to override, but has `observableParameters` = `scaling_IL13_cell`

**Comment by paulstapor** [comment]
This model has observable dependent noise Parameters. Has to be fixed in AMICI, see ICB-DCM/AMICI#609

**Comment by dweindl** [comment]
In the parameter table there are scaling parameters suffixed with '_obs', e.g. `scale_pIL4Ra_obs`. In the measurement file they are not (e.g. scaling_pIL4Ra)

**Comment by LeonardSchmiester** [comment]
fixed by commit f2c64bb7e23093a438d16f6be7ecd2700bca87bf

**Comment by erikadudki** [comment]
While adding the visualization table and comparing the visualization to the publication, it occurs that some datasets are missing (in the measurement and simulation files) ? Is this on purpose?
e.g. For 
- pIL4Ra, pJAK2, pSTAT5: data1, data4
- RecSurf: data2
- SOCS3: data3
- CD274mRNA: data2, data4

Furthermore, for pIL4Ra the simulation for model1_data2 quite underestimates the data, don't know if we want to have a look into that (but I think in the publication this case is similarly underestimated)
![Screenshot from 2020-02-14 09-58-16](https://user-images.githubusercontent.com/49193148/74516379-9076d780-4f10-11ea-822d-ea6b1e46ec9d.png)
